static/js/admin.js

```diff
@@ -588,6 +594,13 @@ function _setup_page() {
                         ui.report_success(i18n.t("Any user may now create new streams!"), create_stream_by_admins_only_status);
                     }
                 }
+                if (response_data.add_emoji_by_admins_only !== undefined) {
```

Comment:
I was guided by previous code structure. I think it's not mandatory, but explicit